docs/research/rhubarb_test/src/index.ts

```diff
@@ -0,0 +1,41 @@
+import { exec } from 'node:child_process'
+import { stdout, chdir, cwd } from 'node:process';
+
+console.log("Start Rhubard....")
+
+console.log(`Starting directory: $${cwd()}`)
+
+cwd()
+
```

Comment:
when we integrate it further later on, we'll probably want to save the current working directory before changing it. That way, we can go back to where we were afterwards.